webpack.config.js

```diff
@@ -135,3 +147,14 @@ module.exports = (env, { mode }) => {
     target: ['web', 'es5'],
   };
 };
+
+async function createJavascriptUiFrameworkFilesWithReplacedPrefix() {
+  await exec('npx tsc'); // run the typescript compiler to generate the dist-tsc folder
```

Comment:
is this only required for creating the output folder? If so, could we not just exec `mkdir -p <OUTPUT_ROOT_DIRECTORY>`?

Reply:
Yes, that sounds good :)